### ISSUE TITLE
stream adventure - websockets 

### ISSUE THREAD

**Issue by anupamkumar** [nodeschool/discussions#873]
What am supposed to see when I open the browser ? I am think I should see "hello" ? But I don't. But still the solution passes ? 

I am confused about what is happening here.

**Comment by khalifenizar** [comment]
Hey there!

The idea is that you write some WebSocket code for the *browser* that sends the *"hello"* message to a local server that `stream-adventure` creates. The server code would then verify that the message is being received as the solution.

That said, I don't think it's currently working as originally intended. Right now `stream-adventure` isn't verifying anything when correcting the WEBSOCKETS exercise. I literally provided it with an empty file and it still passed:

```bash
rm empty-file.js
touch empty-file.js
stream-adventure verify empty-file.js
```

This issue has been reported in the `stream-adventure` repo in the following issues:
- substack/stream-adventure#53
- substack/stream-adventure#78
- substack/stream-adventure#88

Wish I had a better answer. :(

**Comment by anupamkumar** [comment]
Gotcha. Thanks. Hope we get it fixed soon. I ll see if I can do something to help you guys with this. No promises though :)

**Comment by substack** [comment]
fixed in 4.0.0
